zoomage_report.CURATED.tsv

```diff
@@ -56187,3 +56187,215 @@ strain	mixed C57BL/6J and SV129	http://www.ebi.ac.uk/efo/EFO_0000601	E-TABM-426
+inferred cell type - authors labels	lymph node endothelial cell	http://purl.obolibrary.org/obo/CL_0002138	E-EHCA-2	22028_2_203	endothelial cell of lymphatic vessel	mismatch
+inferred cell type - authors labels	mantle B cell	http://www.ebi.ac.uk/efo/EFO_1001468	E-HCAD-1	HCATisStab7587209_S1_L001-CTCGTCAGTCCAGTTA	mantle cell	mismatch
+inferred cell type - authors labels	monocyte progenitor	http://purl.obolibrary.org/obo/CL_0000557	E-HCAD-6	group9-AGGGTGATCTCACATT	granulocyte monocyte progenitor cell	mismatch
+inferred cell type - authors labels	myofibroblast	http://purl.obolibrary.org/obo/CL_0000186	E-HCAD-11	group3-CTCCTAGCAGGTGGAT	myofibroblast cell	mismatch
+inferred cell type - authors labels	myofibroblast	http://purl.obolibrary.org/obo/CL_0000186	E-MTAB-8410	SAMEA6057391-CTAGCCTTCTTGAGAC	myofibroblast cell	mismatch
+inferred cell type - authors labels	naÃ¯ve CD4+ T cell	http://purl.obolibrary.org/obo/CL_0000895	E-HCAD-1	HCATisStabAug177376567_S1_L001-CTAATGGCACTCAGGC	naive thymus-derived CD4-positive, alpha-beta T cell	mismatch
+inferred cell type - authors labels	naive CD8+ T cell	http://purl.obolibrary.org/obo/CL_0000900	E-HCAD-4	group106-GTAGTCATCTCTGAGA	naive thymus-derived CD8-positive, alpha-beta T cell	mismatch
+inferred cell type - authors labels	non-hair root epidermal cell	http://purl.obolibrary.org/obo/PO_0000263	E-GEOD-141730	SAMN13520734-TGCTGCTCAGGAATCG	non-hair root epidermal cell	mismatch
+inferred cell type - authors labels	olfactory ensheathing cell	http://purl.obolibrary.org/obo/CL_0011028	E-GEOD-148360	SRR11518297-GCAGCAATATTA	olfactory ensheathing cell	mismatch
+inferred cell type - authors labels	olfactory horizontal basal cell	http://purl.obolibrary.org/obo/CL_0002169	E-GEOD-151346	SRR11862677-CCTGGTGTATCC	basal cell of olfactory epithelium	mismatch
+inferred cell type - authors labels	pDC	http://purl.obolibrary.org/obo/CL_0000784	E-CURD-46	SAMN12356588-GAGCAGAAGGACAGAA	plasmacytoid dendritic cell	mismatch
+inferred cell type - authors labels	periportal liver sinusoidal endothelial cell	http://www.ebi.ac.uk/efo/EFO_0010705	E-HCAD-9	group4-TCTCATATCGCCTGTT	periportal hepatic sinusoidal endothelial cell	mismatch
+inferred cell type - authors labels	phagocyte	http://purl.obolibrary.org/obo/CL_0000234	E-HCAD-35	SRR9123042-GGTGCGTTCTAGAGTC	phagocyte	mismatch
+inferred cell type - authors labels	phloem pole pericycle	http://purl.obolibrary.org/obo/PO_0000065	E-GEOD-158761	SAMN16293297-TTTGGTTGTCCGACGT	phloem pole pericycle cell	mismatch
+inferred cell type - authors labels	pith	http://purl.obolibrary.org/obo/PO_0006109	E-ENAD-51	SRR12276805-CTAACTTGTCTCTTAT	pith	mismatch
+inferred cell type - authors labels	placental extravillous trophoblast cell (EVT)	http://purl.obolibrary.org/obo/CL_0008036	E-MTAB-6701	ERR2743662-GGACAAGAGTACGTAA	extravillous trophoblast	mismatch
+inferred cell type - authors labels	PSC cell	http://purl.obolibrary.org/obo/CL_0002410	E-MTAB-5061	ERR1630080	pancreatic stellate cell	mismatch
```

Comment:
PSC could also stand for "pluripotent stem cell"

Reply:
OK so with the PSC cell - I also thought pluripotent stem cell but it comes from a pancreas data set and looking at the submission (https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/) I felt it was more likely to be pancreas stellate cell?

Reply:
Yes, you're right. It is that in this particular case. But the mapping is general and doesn't know about this context. If you left that in, next time someone annotates "PSC cell" and means pluripotent stem cell, it automatically gets (wrongly) mapped to stellate cell. Not that we should annotate with acronyms any how ;) ... but to illustrate the idea.

Reply:
Right! OK I see. I will leave it out :)